pkg/features/features.go

```diff
@@ -19,13 +19,14 @@ func init() {
 }
 
 const (
-	// ProcessStatusSync supports synchronization between process message channel statuses
-	ProcessStatusSync featuregate.Feature = "ProcessStatusSync"
+	// RequireAuthorization Supports application access authorization from edge sides
+	// alpha: v1.12
+	RequireAuthorization featuregate.Feature = "requireAuthorization"
 )
 
 // defaultFeatureGates consists of all known Kubeedge-specific feature keys.
 // To add a new feature, define a key for it above and add it here. The features will be
 // available throughout Kubeedge binaries.
 var defaultFeatureGates = map[featuregate.Feature]featuregate.FeatureSpec{
-	ProcessStatusSync: {Default: false, PreRelease: featuregate.Alpha},
+	RequireAuthorization: {Default: true, PreRelease: featuregate.Alpha},
```

Comment:
this is strange, it is a alpha feature but the default value is true

Reply:
Agreed, default value should be false? If users want to use it, just set it in config file

Reply:
done